packages/menu/src/SubMenu/SubMenu.tsx

```diff
@@ -153,15 +151,12 @@ export const SubMenu = InferredPolymorphic<SubMenuProps, 'button'>(
       'aria-disabled': disabled,
       // only add a disabled prop if not an anchor
       ...(typeof rest.href !== 'string' && { disabled }),
+      ...(isAnchor && {
```

Comment:
nit: we can do here what you did with submenu

Reply:
double nit: these components should probably be unified :O

Reply:
Yes they should be... but that's out of scope for this work :upside-down-face: